src/components/Field/Field.js

```diff
@@ -49,19 +56,40 @@ if (tools.shovel) {
 }
 
 export const isInHoverRange = ({
+  fieldMode,
   hoveredPlotRangeSize,
   hoveredPlot: { x: hoveredPlotX, y: hoveredPlotY },
+  itemsSold,
   x,
   y,
 }) => {
   // If hoveredPlotX is null, assume that hoveredPlotY is as well.
-  if (hoveredPlotX == null) {
+  // If fieldMode === OBSERVE, nothing is in hover range.
+  if (hoveredPlotX == null || fieldMode === OBSERVE) {
```

Comment:
what do you think about dropping the comments and changing this to a one liner. i think it would be:

`return hoveredPlotX != null && fieldMode !== OBSERVE`

then just reading the method and conditions tells me that fieldMode of 'observe' will never be in hover range.

Reply:
That would prevent the rest of the code in this function from running, which is the real crux of this function. This initial check just bails out in a few specific cases where the rest of the logic doesn't apply. So, I think this logic needs to stay as-is.

Let me know if I'm understanding your suggestion correctly!